### PR TITLE
Add profile itzwow

### PR DESCRIPTION
**Pull request by itzwow**
## Check List (Check all the applicable boxes) 

- [ x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

-> Add a new profile itzwow

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1130"></a>

### CODE REVIEW

File: public/data/itzwow.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "AMIT KUMAR BERA",
+    "type": "personal",
+    "bio": "Web Developer, Open Source Enthusiast! & Explorer",
```
Comment (syedsohan) [inline public/data/itzwow.json:4]:
```suggestion
    "bio": "Web Developer, Open Source Enthusiast & Explorer!",
```

File: public/data/itzwow.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/itzwow",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/itzyourab",
+        "icon": "twitter"
+      },{
```
Comment (syedsohan) [inline public/data/itzwow.json:16]:
```suggestion
      },
      {
```

File: public/data/itzwow.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        "name": "Follow me on Linkedin",
```
Comment (syedsohan) [inline public/data/itzwow.json:17]:
```suggestion
        "name": "Follow me on LinkedIn",
```

File: public/data/itzwow.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        "url": "https://www.linkedin.com/in/amit-kumar-bera-990954205/",
+        "icon": "linkedin"
+      }
+    ]
+}
```
Comment (syedsohan) [inline public/data/itzwow.json:22]:
```suggestion
}

```

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/itzwow.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        "url": "https://github.com/itzwow",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/itzyourab",
+        "icon": "twitter"
+      },{
+        "name": "Follow me on Linkedin",
+        "url": "https://www.linkedin.com/in/amit-kumar-bera-990954205/",
```
Comment (KendallDoesCoding) [inline public/data/itzwow.json:18]:
```suggestion
        "url": "https://www.linkedin.com/in/amit-kumar-bera-990954205",
```

The / is not needed in the url, it works anyway. It just gets added when you copy the url from the taskbar.

File: public/data/itzwow.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "AMIT KUMAR BERA",
```
Comment (KendallDoesCoding) [inline public/data/itzwow.json:2]:
```suggestion
    "name": "Amit Kumar Bera",
```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: public/data/itzwow.json (hunk 2 above)
Comment (KendallDoesCoding) [reply public/data/itzwow.json:16]: Github-actions will handle this once the PR is merged so dw about it @syedsohan

File: public/data/itzwow.json (hunk 4 above)
Comment (KendallDoesCoding) [reply public/data/itzwow.json:22]: Github-actions will handle this once the PR is merged so dw about it @syedsohan

Comment (eddiejaoude) [reply public/data/itzwow.json:22]: It is better people learn and fix what they need, not all projects will have the same GitHub Action - plus it doesn't make their github activity look good

File: public/data/itzwow.json (hunk 5 above)
Comment (eddiejaoude) [reply public/data/itzwow.json:18]: As you mentioned this will still work, I think these types of feedback are unnecessary notifications

**Comment by KendallDoesCoding** [comment]
Yup true. People should learn how to format the code without relying on
github-actions always.

**Comment by KendallDoesCoding** [comment]
Yup. I thought it would be better for formatting reasons but many people
seem to be saying its unnecessary and yeah it is unnecessary notifics.
Won't suggest changes like this in future.

**Review by Panquesito7 — APPROVED** [review]

**Review by ssensalo — APPROVED** [review]

**Pull request merged**